### ISSUE TITLE
Table doesn't have style when using AllStyledComponent

### ISSUE THREAD

**Issue by ocavue** [remirror/remirror#878]
### Summary

When using `<AllStyledComponent/>`, the table seems don't have any style.

### Steps to reproduce

```
$ git clone https://github.com/issueset/remirror-table-style
$ cd remirror-table-style
$ pnpm i 
$ pnpm run start 
```

Open localhost:3000 and press the button "insert a 3*3 table".

### Expected results

The table is inserted and has style (like borders).

### Actual results

The table is inserted and doesn't have any style.

### Possible Solution

N/A 

### Screenshot(s)

![image](https://user-images.githubusercontent.com/24715727/109164316-3e1ff600-77b5-11eb-8daa-bd4ebdf31018.png)

**Comment by ocavue** [comment]
Just found out that `extensionTablesStyledCss` doesn't have a border style, which means that I can't tell if `AllStyledComponent` works by the border. 😂

**Comment by ifiokjr** [comment]
This was fixed in the beta branch here [`967da30` (#706)](https://github.com/remirror/remirror/pull/706/commits/967da3019aa868f22fc5637253da331226ef818c)
